make/common/NativeCompilation.gmk

```diff
@@ -508,8 +524,7 @@ define SetupNativeCompilation
               $1 += $$(subst $$($1_OBJECT_DIR),$$($1_OUTPUT_DIR),$$($1_DEBUGINFO_FILES))
             endif
           endif
-        endif
-      endif # !MacOS X
+      endif # ENABLE_DEBUG_SYMBOLS
```

Comment:
Something wrong about the whitespace (4 vs 2) here which makes me wonder if we've missed indenting/scoping above

Reply:
yes, I didn't want to change the whole block indent, but I can do that